examples/basic/test.js

```diff
@@ -83,7 +83,7 @@ test('Moving the slider', async t => {
 
     await t
         .click(page.triedTestCafeCheckbox)
-        .dragToElement(page.slider.handle, page.slider.tick.with({ text: '9' }));
+        .dragToElement(page.slider.handle, page.slider.tick.withText('9'));
```

Comment:
Also, I guess we should redesign this page model to be a class (so we'll be able to create selector derivatives). E.g. we can create single label selector: `const label = Selector('label')` and then expose other selectors as derivatives: `label: label.withText('Support for testing on remote devices')`. `interfaceSelectOption` can be a derivative of `interfaceSelect`: `interfaceSelect.find('option')`